### PR TITLE
V5

### PR DESCRIPTION
**Pull request by mongjong59**
@justin808, @robwise, @alexfedoseev

### CODE REVIEW

**Comment by justin808** [comment]
@robwise @alexfedoseev Please let me know if this PR addresses your prior comments. I will take a look as well.

**Comment by justin808** [comment]
@alleycat-at-git @robwise @alexfedoseev I think it's worth considering a further simplification of having one file with reducer, constants, actions.

## Redux Ducks?

https://github.com/erikras/ducks-modular-redux

> Ducks: Redux Reducer Bundles
> 
> I find as I am building my redux app, one piece of functionality at a time, I keep needing to add {actionTypes, actions, reducer} tuples for each use case. I have been keeping these in separate files and even separate folders, however 95% of the time, it's only one reducer/actions pair that ever needs their associated actions.
> 
> To me, it makes more sense for these pieces to be bundled together in an isolated module that is self contained, and can even be packaged easily into a library.

## Or Stories

https://github.com/shakacode/redux-stories

File: client/package.json [hunk 1]
```diff
@@ -33,5 +36,10 @@
     "webpack": "^1.12.14"
   },
   "devDependencies": {
+    "eslint": "^3.10.2",
+    "eslint-config-shakacode": "^13.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^6.7.1"
```
Comment (robwise) [inline client/package.json:43]: @justin808 we agreed a long time ago that we should not put linters in the generators

Comment (justin808) [reply client/package.json:43]: @robwise They are not in the generator. Are they @nostophilia? 

@nostophilia was instructed to use the linter to be sure what we generated would run clean.

Comment (mongjong59) [reply client/package.json:43]: @justin808 @robwise 
No, they aren't. Linters are added after running the generator. Code generated by generator is within the first commit.

Comment (robwise) [reply client/package.json:43]: Oh okay nvm then

**Pull request closed without merging**